distribution/docs/src/main/resources/_contents/_endpoints/cometd-endpoint-contents.adoc

```diff
@@ -0,0 +1,845 @@
+
+==== CometD Endpoint
+
+The http://cometd.org[CometD] endpoint enables asychronous search capabilties.
+The CometD protocol is used to execute searches, retrieve results, and receive notifications.
+
+For an example of using CometD within a webapp see: `distribution/sdk/sample-cometd/`
+
+===== Installing CometD Endpoint
+
+The CometD Endpoint is installed by default with a standard installation in the ${ddf-ui} application.
+
+===== Configuring CometD Endpoint
+
+None.
+
+===== Using CometD Endpoint
+
+.CometD Endpoint URL
+----
+${secure_url}/search/cometd
+----
+
+====== Query
+
+Queries can be executed over CometD using the `/service/query` channel.
+Query messages are json formatted and use cql alongside several other parameters.
+
+.Query Parameters
+[cols="1,3,1", options="header"]
+|===
+|Parameter Name
+|Description
+|Required
+
+|`src`
+|Catalog Source
+|No
+
+|`cql`
+|CQL query
+|Yes
+
+|`sort`
+|Sort Type
```

Comment:
We should provide more information on the sort, here or after the table.

Basically, the format of the sort option is `<Sort Field>:<Sort Order>`, where `<Sort Order>` is either `asc` or `desc`.